report/src/discussion/discussion.tex

```diff
@@ -0,0 +1,38 @@
+We found that utilizing four beacons to outline a room and a map generated by traversing the room in a grid pattern of $1m \times 1m$ we were able to achieve significant results.
+Depending on the chosen \textit{k} for nearest neighbor search, we were able to achieve between $95.24$ and $97.62$ percent accuracy for the inside case and between $45.33$ and $78.67$ percent accuracy for the outside scenario. 
+As mentioned in \ref{sec:ChoiceOfTechnique}, the goal of the project was not to achieve high accuracy and precision on the exact positioning of people in a room. 
+The goal was rather to investigate whether we could detect the presence of people in a meeting with reasonable certainty.
+
+These findings align with what similar research has concluded; that using kNN is sufficiently accurate, and that complex machine learning is not necessary to obtain useful results.\cite{ble_kneares_neural}
+
+The experimental setup depicted in Figure~\ref{fig:experiment_setup} places four beacons within a room.
+Despite the constraints of time and budget, the setup yielded valuable data and insights.
+
+Looking towards future investigations, the placement of beacons outside the room would be worth investigating.
+Such a configuration could potentially impact the predictive accuracy of the model.
+Similarly, varying the total number of beacons, both inside and outside the room, could impact data richness and consequently, the precision of predictions.
+
+Furthermore, extending the grid points within and beyond the room could also yield benefits. By adjusting the granularity of data collected, we may observe a corresponding change in the precision of the predictive model. 
+
+However, it is worth noticing that for measuring points \textit{B} and \textit{J} in figure \ref{fig:experiment_room}, a large uncertainty in classification is observed.
+We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it.
+This low absorbation results in the two data points next to the pane having similar distances to both map values labelled as \textit{inside} and some as \textit{outside}, therefore giving uncertain classifications.
+
+A limitation with the presented approach is that it requires employees to carry a device capable of running the application in the background.
+This makes the approach unsuitable for organizations that do not provide their employees with such devices.
+It is a tradeoff between convenience and accuracy.
+Proximity detection, for example, would have been more convinient, since it would only require employees to carry a Bluetooth-enabled devices (such as a mobile phone or Bluetooth tag), however this method is far less accurate, as discussed in Section~\ref{sec:proximity_detection}.
```

Comment:
```suggestion
Proximity detection, for example, would have been more convinient, since it would only require employees to carry a Bluetooth-enabled devices (such as a mobile phone or Bluetooth tag). However, this method is far less accurate, as discussed in Section~\ref{sec:proximity_detection}.
```